2021-10-31 10:33:01,257 INFO    MainThread:664 [wandb_setup.py:_flush():71] setting env: {}
2021-10-31 10:33:01,258 INFO    MainThread:664 [wandb_setup.py:_flush():71] setting login settings: {}
2021-10-31 10:33:01,258 INFO    MainThread:664 [wandb_setup.py:_flush():71] setting login settings: {}
2021-10-31 10:33:01,258 INFO    MainThread:664 [wandb_init.py:_log_setup():357] Logging user logs to /home/ubuntu/w251-hw9/master/wandb/run-20211031_103301-2l0kj1c5/logs/debug.log
2021-10-31 10:33:01,258 INFO    MainThread:664 [wandb_init.py:_log_setup():358] Logging internal logs to /home/ubuntu/w251-hw9/master/wandb/run-20211031_103301-2l0kj1c5/logs/debug-internal.log
2021-10-31 10:33:01,258 INFO    MainThread:664 [wandb_init.py:init():390] calling init triggers
2021-10-31 10:33:01,258 INFO    MainThread:664 [wandb_init.py:init():395] wandb.init called with sweep_config: {}
config: {'epochs': 2, 'batch_size': 500, 'momentum': 0.9, 'WEIGHT_DECAY': 0.0001, 'arch': 'resnet18'}
2021-10-31 10:33:01,258 INFO    MainThread:664 [wandb_init.py:init():410] re-initializing run, found existing run on stack: pscwd9uw
2021-10-31 10:33:01,259 INFO    MainThread:664 [wandb_run.py:finish():1347] finishing run rochelleli/w251-hw9/pscwd9uw
2021-10-31 10:33:01,259 INFO    MainThread:664 [jupyter.py:save_history():429] not saving jupyter history
2021-10-31 10:33:01,259 INFO    MainThread:664 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-10-31 10:33:01,259 INFO    MainThread:664 [wandb_init.py:_jupyter_teardown():289] cleaning up jupyter logic
2021-10-31 10:33:01,260 INFO    MainThread:664 [wandb_run.py:_atexit_cleanup():1663] got exitcode: 0
2021-10-31 10:33:01,261 INFO    MainThread:664 [wandb_run.py:_restore():1635] restore
2021-10-31 10:33:01,648 INFO    MainThread:664 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 783
  total_bytes: 783
}

2021-10-31 10:33:01,766 INFO    MainThread:664 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 783
  total_bytes: 783
}

2021-10-31 10:33:02,108 INFO    MainThread:664 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 783
  total_bytes: 5154
}

2021-10-31 10:33:02,210 INFO    MainThread:664 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 5154
  total_bytes: 5154
}

2021-10-31 10:33:02,310 INFO    MainThread:664 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 5154
  total_bytes: 5154
}

2021-10-31 10:33:02,411 INFO    MainThread:664 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 5154
  total_bytes: 5154
}

2021-10-31 10:33:02,512 INFO    MainThread:664 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 5154
  total_bytes: 5154
}

2021-10-31 10:33:02,613 INFO    MainThread:664 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 5154
  total_bytes: 5154
}

2021-10-31 10:33:02,754 INFO    MainThread:664 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 5154
  total_bytes: 5154
}

2021-10-31 10:33:02,955 INFO    MainThread:664 [wandb_run.py:_wait_for_finish():1793] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 5154
  total_bytes: 5154
}
local_info {
}

2021-10-31 10:33:04,059 INFO    MainThread:664 [wandb_run.py:_append_files():2061] logging synced files
2021-10-31 10:33:04,062 INFO    MainThread:664 [wandb_init.py:init():435] starting backend
2021-10-31 10:33:04,062 INFO    MainThread:664 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-10-31 10:33:04,066 INFO    MainThread:664 [backend.py:ensure_launched():199] starting backend process...
2021-10-31 10:33:04,070 INFO    MainThread:664 [backend.py:ensure_launched():204] started backend process with pid: 746
2021-10-31 10:33:04,071 INFO    MainThread:664 [wandb_init.py:init():444] backend started and connected
2021-10-31 10:33:04,073 INFO    MainThread:664 [wandb_run.py:_label_probe_notebook():866] probe notebook
2021-10-31 10:33:04,073 INFO    MainThread:664 [wandb_run.py:_label_probe_notebook():876] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2021-10-31 10:33:04,073 INFO    MainThread:664 [wandb_init.py:init():503] updated telemetry
2021-10-31 10:33:04,083 INFO    MainThread:664 [wandb_init.py:init():533] communicating current version
2021-10-31 10:33:04,788 INFO    MainThread:664 [wandb_init.py:init():538] got version response 
2021-10-31 10:33:04,788 INFO    MainThread:664 [wandb_init.py:init():548] communicating run to backend with 30 second timeout
2021-10-31 10:33:04,894 INFO    MainThread:664 [wandb_init.py:init():576] starting run threads in backend
2021-10-31 10:33:05,397 INFO    MainThread:664 [wandb_run.py:_console_start():1693] atexit reg
2021-10-31 10:33:05,397 INFO    MainThread:664 [wandb_run.py:_redirect():1567] redirect: SettingsConsole.WRAP
2021-10-31 10:33:05,397 INFO    MainThread:664 [wandb_run.py:_redirect():1604] Wrapping output streams.
2021-10-31 10:33:05,398 INFO    MainThread:664 [wandb_run.py:_redirect():1628] Redirects installed.
2021-10-31 10:33:05,398 INFO    MainThread:664 [wandb_init.py:init():603] run started, returning control to user process
